src/Workspaces/Core/Portable/Workspace/Solution/StateChecksums.cs

```diff
@@ -257,33 +257,6 @@ public static SolutionStateChecksums Deserialize(ObjectReader reader)
             ChecksumCollection.Find(solution.AnalyzerReferences, AnalyzerReferences, searchingChecksumsLeft, result, cancellationToken);
         }
 
-        if (assetPath.IncludeTopLevelProjects)
```

Comment:
this now falls out by passing `AssetPathKind.Projects` but not passing in a specific project-id